### DISCUSSION TITLE
lang-specific `editor` settings?

### DISCUSSION THREAD

**Discussion by Rudxain** [helix-editor/helix#10780] [Q&A]
Is it possible to add some settings from `config.toml` to `languages.toml`, such that they only apply to a given lang? For example, let markdown be the only lang where `soft-wrap` is enabled by default

**Comment by TornaxO7** [comment]
As far as I know this isn't possible at the moment. But `soft-wrap` can be set in the `language.toml`.

**Reply by Rudxain** [reply]
oh :( . Well, I guess that's good enough

**Reply by TornaxO7** [reply]
What you could do is to create a `config.toml` per project.

**Reply by Rudxain** [reply]
> per project.

currently, I don't have the need to do that, but thanks for the suggestion anyways!
